inst/extdata/PA2022CH_en_exec_summary/template.Rmd

```diff
@@ -159,11 +159,11 @@ data_scatter_power_b <- prep_scatter(
   scenario_selected = scenario_selected,
   asset_class = "bonds"
 )
- plot_scatter(data_scatter_power_b) +
-   patchwork::plot_annotation(
-     title = "Bonds",
-     theme = theme(plot.title = element_text(face = "bold", hjust = 0.5, size = 22))
-     )
+ # plot_scatter(data_scatter_power_b) +
+ #   patchwork::plot_annotation(
+ #     title = "Bonds",
+ #     theme = theme(plot.title = element_text(face = "bold", hjust = 0.5, size = 22))
+ #     )
```

Comment:
sorry to not comment before this was merged...

Is there a good reason to leave these in commented out, or can we just get rid of it?

Reply:
I mean for now I just have them commented out because the scatterplots are still not consistently working... I'm not sure about the plan for those, but I would expect they come back at some point today when the `guide_area()` issue is fixed @cjyetman @MonikaFu